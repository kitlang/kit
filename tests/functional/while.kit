function main() {
    var a = 1;
    while (a < 5) {
        printf("%i\n", a);
        a++;
    }

    var b = 1;
    while (true) {
        if b == 2 {
            ++b;
            continue;
        }
        printf("%i\n", b);
        if ++b == 4 {
            break;
        }
    }

    var c = 3;
    do {
        printf("at least once\n");
    } while (c < 3);
}
