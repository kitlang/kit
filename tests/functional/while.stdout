1
2
3
4
1
3
at least once
